appgate/appgate.py

```diff
@@ -401,40 +312,67 @@ async def operator(
             )
             if plan.needs_apply:
                 log.info(
-                    "[appgate-operator/%s] No more events for a while, creating a plan",
+                    "[%s/%s] No more events for a while, creating a plan",
+                    operator_name,
                     namespace,
                 )
-                new_plan = await appgate_plan_apply(
+                entity_clients = None
+                if not ctx.dry_run_mode and not ctx.reverse_mode:
+                    entity_clients = generate_api_spec_clients(
+                        api_spec=ctx.api_spec, appgate_client=appgate_client
+                    )
+                if not ctx.dry_run_mode and ctx.reverse_mode:
+                    entity_clients = generate_k8s_clients(
+                        api_spec=ctx.api_spec,
+                        namespace=ctx.namespace,
+                        k8s_api=get_crds(),
+                    )
```

Comment:
Interesting way of implementing the dry run.  Took me a bit to understand but it makes sense. If the entity_clients is None, `plan_apply` doesn't run any of the entity_client's action.